readthedocs/settings/docker_compose.py

```diff
@@ -18,7 +18,6 @@ class DockerBaseSettings(CommunityDevSettings):
     PRODUCTION_DOMAIN = 'community.dev.readthedocs.io'
     PUBLIC_DOMAIN = 'community.dev.readthedocs.io'
     PUBLIC_API_URL = f'http://{PRODUCTION_DOMAIN}'
-    RTD_PROXIED_API_URL = PUBLIC_API_URL
```

Comment:
Shouldn't we keep the setting, to make it easier for users to override? I think supporting the setting is fine, I just want a default that is `_/` as we have now.

Reply:
We had that setting to pass it down to context during build time. But we want to apply this change to everyone. So, not sure if we can have both.